src/panels/developer-tools/statistics/developer-tools-statistics.ts

```diff
@@ -117,26 +117,26 @@ class HaPanelDevStatistics extends SubscribeMixin(LitElement) {
       actions: {
         title: "",
         type: "overflow-menu",
-        template: (
-          _info,
-          statistic: StatisticsMetaData
-        ) => html`<ha-icon-overflow-menu
-          .hass=${this.hass}
-          .narrow=${this.narrow}
-          .items=${[
-            {
-              path: mdiSlopeUphill,
-              label: localize(
-                "ui.panel.developer-tools.tabs.statistics.adjust_sum"
-              ),
-              action: () =>
-                showStatisticsAdjustSumDialog(this, {
-                  statistic: statistic,
-                }),
-            },
-          ]}
-          style="color: var(--secondary-text-color)"
-        ></ha-icon-overflow-menu>`,
+        template: (_info, statistic: StatisticsMetaData) =>
+          statistic.has_sum
+            ? html`<ha-icon-overflow-menu
```

Comment:
Why Overflow for one icon?

Reply:
Honestly when I was initially hacking on it because I have no idea how to just render a single icon in the data overflow table and get the CSS right

Reply:
Im hacking on it :)